src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -16,8 +16,13 @@ import PAYMENT_METHOD_INFO from '@salesforce/schema/DataImport__c.Payment_Method
 import ACCOUNT1_IMPORTED_INFO from '@salesforce/schema/DataImport__c.Account1Imported__c';
 import CONTACT1_IMPORTED_INFO from '@salesforce/schema/DataImport__c.Contact1Imported__c';
 import DONATION_DONOR_INFO from '@salesforce/schema/DataImport__c.Donation_Donor__c';
+import CONTACT1_LASTNAME_INFO from '@salesforce/schema/DataImport__c.Contact1_Lastname__c';
+import ACCOUNT1_NAME_INFO from '@salesforce/schema/DataImport__c.Account1_Name__c';
 
 const WARNING = 'warning';
+const DONATION_DONOR_LABEL = 'Donation Donor';
+const REQUIRED_FORM_FIELDS_MESSAGE = 'Account1 Imported, Account1 Name, Contact1 Imported, Contact1 Last Name';
```

Comment:
Can we grab these field labels from the imported schema information (ACCOUNT1_IMPORTED_INFO, etc) instead of hard-coding the strings? Same with "Donation Donor" above?

Reply:
It doesn't seem like we can retrieve the field label through the schema, the variable returned from the schema call only carries the objectApiName and fieldApiName. Except if we want to do some string manipulation on the fieldApiName, although we won't be able to guarantee that they will be the same(fieldApiName and field label).

Reply:
I take back my comment, found a way around it. Thanks for calling it out!